packages/web/pages/api/edge-trpc-assets/[trpc].ts

```diff
@@ -0,0 +1,32 @@
+import { fetchRequestHandler } from "@trpc/server/adapters/fetch";
+import { NextRequest } from "next/server";
+
+import { edgeRouter } from "~/server/api/edge-routers/edge-router";
+import { createEdgeTRPCContext } from "~/server/api/trpc";
+import { constructEdgeUrlPathname } from "~/utils/trpc-edge";
+
+// We're using the edge-runtime
+export const config = {
+  runtime: "edge",
```

Comment:
Thoughts on serverless, since we determined it was a memory issue and not event loop queue issue? There are issues with edge, even without load

Reply:
I think this change should be made in a separate PR because it requires some considerations:

1. Our node functions are in just one region, so we need a plan to manage the extra load on the SQS server cc @p0mvn 
2. Based on benchmarks, this change adds ~1.5s to every query, on top of the initial ~10s needed for a cold start
3. We must rename our edge router, as keeping its current name would wrongly suggest that these routes run on the edge when they run on a node

Reply:
1. Is there no way to make node multi region
2. I think we're still having issues in Asia, I 
3. What if we remove `edge` entirely from our tRPC router namespaces? With how tRPC lets us run logic to determine a route's link, I think there's no reason to have the runtime be indicated by the query path. Rather, we can just organize routes by data type or feature